ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/cv-pathfinder-edge-custom-default-policy.yml

```diff
@@ -171,16 +171,16 @@ agents:
 flow_tracking:
   hardware:
     trackers:
-    - name: custom_flow_track_name
+    - name: FLOW-TRACKER
```

Comment:
Looks like this changes - is it not a problem?

Reply:
The AvdStructuredConfig changes were not made in this patch. Resolved now.